pkg/gits/bitbucket_server_test.go

```diff
@@ -50,9 +50,21 @@ var bitbucketServerRouter = util.Router{
 		"GET": "pr.json",
 		"PUT": "pr.json",
 	},
+	"/rest/api/1.0/projects/TEST-ORG/repos/test-repo/pull-requests/7": util.MethodMap{
```

Comment:
Orgs are always capitalized in BitBucket? Maybe we can add a link to the API docs or something. What happens if the org is not capitalized?

Reply:
Like I said above, the org (well, project in BBS terminology) is always capitalized in BBS. I had a BDD failure trying to call with `jxbdd` rather than `JXBDD`, e.g.